class.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'module'
```

Comment:
- [ ] Please make sure `require_relative 'my_enumerable'` is present here instead of module.